jwql/website/apps/jwql/static/js/jwql.js

```diff
@@ -6,8 +6,93 @@
  * @author Brad Sappington
  * @author Bryan Hilbert
  * @author Maria Pena-Guerrero
+ * @author Melanie Clarke
  */
 
+
+ /**
+ * Change the filetype for all displayed images
+ * @param {String} type - The image type (e.g. "rate", "uncal", etc.)
+ * @param {String} group_root - The rootname of the file group
+ * @param {Dict} num_ints - A dictionary whose keys are suffix types and whose
+ *                          values are the number of integrations with an associated
+ *                          preview image for that suffix
+ * @param {Dict} available_ints - A dictionary whose keys are suffix types and whose
+ *                                values are the integration numbers of the available
+ *                                jpgs for that suffix
+ * @param {Dict} total_ints - A dictionary whose keys are suffix types and whose
+ *                                values are the total number of integrations for that
+ *                                filetype.
+ * @param {String} inst - The instrument for the given file
+ */
+ function change_all_filetypes(type, group_root, num_ints, available_ints, total_ints, inst, detectors) {
+
+    // Change the radio button to check the right filetype
+    document.getElementById(type).checked = true;
+
+    // Store the currently displayed suffix
+    document.getElementById("view_file_type").setAttribute('data-current-suffix', type);
+
+    // Clean the input parameters
+    num_ints = num_ints.replace(/&#39;/g, '"');
+    num_ints = num_ints.replace(/'/g, '"');
```

Comment:
These two cleaning lines are used a lot of places.  Consider making a function in case we ever need to expand on it or use it elsewhere? 

```suggestion
    num_ints = clean_int_param(num_ints);

    function clean_int_param(int_to_clean){
        int_to_clean = int_to_clean.replace(/&#39;/g, '"');
        int_to_clean = int_to_clean.replace(/'/g, '"');
        return int_to_clean;
    } 
```

Reply:
Sure, I can add that.

Reply:
Done!